packages/nitro-protocol/gas-benchmarks/fixtures.ts

```diff
@@ -169,41 +183,72 @@ export const Y = new TestChannel(
   3,
   [Alice, Bob],
   [
-    {destination: convertAddressToBytes32(Alice.address), amount: amountForAlice},
-    {destination: convertAddressToBytes32(Bob.address), amount: amountForBob},
+    {
+      destination: convertAddressToBytes32(Alice.address),
+      amount: amountForAlice,
+      metadata: '0x',
+      allocationType: AllocationType.simple,
+    },
+    {
+      destination: convertAddressToBytes32(Bob.address),
+      amount: amountForBob,
+      metadata: '0x',
+      allocationType: AllocationType.simple,
+    },
   ]
 );
 
 /** Ledger channel between Alice and Bob, providing funds to channel X */
 export const LforX = new TestChannel(
   4,
   [Alice, Bob],
-  [{destination: X.channelId, amount: amountForAliceAndBob}]
+  [
+    {
+      destination: X.channelId,
+      amount: amountForAliceAndBob,
+      metadata: '0x',
+      allocationType: AllocationType.simple,
+    },
+  ]
 );
 
 /** Joint channel between Alice, Bob, and Ingrid, funding application channel X */
 export const J = new TestChannel(
   5,
   [Alice, Bob, Ingrid],
   [
-    {destination: X.channelId, amount: amountForAliceAndBob},
-    {destination: convertAddressToBytes32(Ingrid.address), amount: amountForAliceAndBob},
+    {
+      destination: X.channelId,
+      amount: amountForAliceAndBob,
+      metadata: '0x',
+      allocationType: AllocationType.simple,
+    },
+    {
+      destination: convertAddressToBytes32(Ingrid.address),
+      amount: amountForAliceAndBob,
+      metadata: '0x',
+      allocationType: AllocationType.simple,
+    },
   ]
 );
 
-/** Guarantor channel between Alice and Ingid, targeting joint channel J */
-export const G = new TestChannel(6, [Alice, Ingrid], {
-  targetChannelId: J.channelId,
-  destinations: [
-    X.channelId,
-    convertAddressToBytes32(Alice.address),
-    convertAddressToBytes32(Ingrid.address),
-  ],
-});
-export const LforG = new TestChannel(
+/** Ledger channel between Alice and Ingid, with Guarantee targeting joint channel J */
+
+export const LforJ = new TestChannel(
   7,
   [Alice, Bob],
-  [{destination: G.channelId, amount: amountForAliceAndBob}]
+  [
+    {
+      destination: J.channelId,
+      amount: '0xa',
```

Comment:
Nit: Could this be a constant value instead of magic string?

Reply:
732a22917f34cb87b82dd467097a4d6cf1bbbcda

Reply:
732a22917f34cb87b82dd467097a4d6cf1bbbcda